app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to new_users_path
+    else
+      render :new
```

Comment:
When i tried to save a new user, it gave me an error `undefined local variable or method 'new_users_path'`. Maybe a typo